resources/core/charts/apiserver-proxy/README.md

```diff
@@ -11,13 +11,13 @@ _  __     _                          _                       _____ _____    ____
 
 ## Overview
 
-This API Server Proxy is an [Nginx](https://nginx.org/en/)-based, transparent proxy for the Kubernetes API. It is exposed for the external communication.
+This API Server Proxy is an [kube-rbac-proxy](https://github.com/brancz/kube-rbac-proxy)-based, transparent proxy for the Kubernetes API. It is exposed for the external communication.
```

Comment:
This API Server Proxy is a transparent proxy for the Kubernetes API based on [kube-rbac-proxy](https://github.com/brancz/kube-rbac-proxy). It is exposed for external communication.